cmd/kured/main.go

```diff
@@ -93,6 +96,10 @@ func init() {
 }
 
 func main() {
+	os.Stdout.WriteString(fmt.Sprintf("Command line args: %v\n", os.Args))
```

Comment:
do we need this line?  Do we need it in main?

Reply:
No, sorry these were to help me figure out an issue related to https://github.com/microsoft/hcsshim/pull/1207.
I'll remove all of these.

Reply:
Well, you found it useful to have flags displayed, so we should probably address that in a later patch :)